modules/openapi-generator/src/test/resources/3_0/python-experimental/petstore-with-fake-endpoints-models-for-testing-with-http-signature.yaml

```diff
@@ -1703,6 +1716,10 @@ components:
         - placed
         - approved
         - delivered
+    ArrayOfEnums:
+      type: array
+      items:
+        $ref: '#/components/schemas/OuterEnum'
```

Comment:
Does the model AnimalFarm already demonstrate your use case?

Reply:
I haven't seen it used as top level object. I can use it here instead. WDYT?

Reply:
What do you mean by top level object? Do you mean used in an endpoint?
Testing shows that the same code path is hit with that model and this one so if you can use that schema and avoid adding this schema it would be preferred because it will keep our sample spec smaller.

Reply:
As I was not sure about the purpose, I have initially created new model. I am using the `AnimalFarm` now.